Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:latest
```

Comment:
Codacy found an issue: [Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag](https://app.codacy.com/manual/tianhaoz/approveman/pullRequest?prid=6227497)